examples/trusted_database/proto/trusted_database.proto

```diff
@@ -30,15 +30,25 @@ message PointOfInterest {
   Location location = 2;
 }
 
+message GetPointOfInterestRequest {
+  string id = 1;
+}
+
+message GetPointOfInterestResponse {
+  PointOfInterest point_of_interest = 1;
+}
+
 message ListPointsOfInterestRequest {
   Location location = 1;
 }
 
 message ListPointsOfInterestResponse {
-  PointOfInterest point_of_interest = 2;
+  PointOfInterest point_of_interest = 1;
 }
 
 service TrustedDatabase {
+  // Find Point Of Interest based on id.
+  rpc GetPointOfInterest(GetPointOfInterestRequest) returns (GetPointOfInterestResponse);
```

Comment:
This seems to have been added just for the sake of the benchmark, and is not part of the actual functionality of a TrustedDatabase. If this is the case, it would be nice to mention it in the comment.

Even better, if it is possible, define it as a separate service, for instance `TrustedDatabaseBenchmark`, and implement the generated trait for `TrustedDatabaseHandlerNode`, and `TrustedDatabaseService ` in the benchmark backend. But I don't know if it is possible to implement two services for `TrustedDatabaseHandlerNode`.